indigo/sandbox/src/main/scala/com/example/sandbox/scenes/PointersScene.scala

```diff
@@ -0,0 +1,100 @@
+package com.example.sandbox.scenes
+
+import com.example.sandbox.Fonts
+import com.example.sandbox.SandboxAssets
+import com.example.sandbox.SandboxGameModel
+import com.example.sandbox.SandboxStartupData
+import com.example.sandbox.SandboxViewModel
+import indigo.*
+import indigo.scenes.*
+import indigo.shared.events.PointerEvent
```

Comment:
Can you add an alias to this to the [package](https://github.com/PurpleKingdomGames/indigo/blob/33a63baefb212fe21f7179783c05b36f930e3e97/indigo/indigo/src/main/scala/indigo/package.scala#L372) so that it just arrives with the standard imports?